crates/subspace-runtime/src/lib.rs

```diff
@@ -251,6 +251,7 @@ impl frame_system::Config for Runtime {
     type PostInherents = ();
     type PostTransactions = ();
     type MaxConsumers = ConstU32<16>;
+    type ExtensionsWeightInfo = ();
```

Comment:
This impl assumes RocksDB, are these the weights we actually want in production?
https://docs.rs/frame-system/latest/src/frame_system/extensions/weights.rs.html#139-209

(This issue happens in almost every runtime change in this PR.)